terraso_backend/apps/graphql/schema/commons.py

```diff
@@ -147,6 +167,12 @@ def mutate_and_get_payload(cls, root, info, **kwargs):
     def is_update(cls, data):
         return "id" in data
 
+    @classmethod
+    def not_allowed(cls, mutation_type=None):
```

Comment:
Oh good point. I think because `BaseAuthenticatedMutation` has `model_class` property, while the other one does not. I think I could at least call the parent method from the child...